handler/instance_started.go

```diff
@@ -290,10 +294,10 @@ func (h *InstanceStarted) TriggerImportDimensionOptions(r *recipe.Recipe, codeli
 
 		h.producer.Channels().Output <- b
 	}
-	if nonGeographyCount == 0 {
+	if geographyCount == tc {
 		var errs []error
 		errs = append(errs, &Error{
-			err:     fmt.Errorf("no non-geography variables exist in the codelists"),
+			err:     fmt.Errorf("only geography codelists exist in this instance"),
```

Comment:
Please change the above string (to aid maintenance) from:
"only geography codelists exist in this instance"
to:
"only geography codelists exist in this instance, there must be at least one non-geography in the codelists"

Reply:
This has been addressed.